css/style.css

```diff
@@ -526,6 +591,56 @@ section {
 }
 
 @media (max-width: 800px) {
+  #navOptions #story-li.active {
```

Comment:
this feels also quite redundant. u should be able to write the following additions in 5-10 lines max